crates/oracle_document/coordinator/src/oracle_document.rs

```diff
@@ -22,7 +22,12 @@ pub fn create_oracle_document(oracle_document: OracleDocument) -> ExternResult<R
 
     Ok(record)
 }
-pub fn get_latest_oracle_document_ah_for_name(
+#[hdk_extern]
+pub fn get_latest_oracle_document_ah_for_name(payload:ExecuteOracleRecipePayload) -> ExternResult<Option<ActionHash>> {
+    get_latest_document_ah_for_name(payload.name, &payload.trusted_authors)
+}
+
+pub fn get_latest_document_ah_for_name(
```

Comment:
What's with the extra wrapper?

Reply:
holoom_recipe -> holoom_recipe_execution ->*n.  (all other zomes). the wrapper allows for the holoom_recipe_execution zome to call the oracle zome (bridge call)